website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -7,6 +7,12 @@ description: "Reference public models across dbt projects"
 
 :::caution Closed Beta - dbt Cloud Enterprise
 "Project" dependencies and cross-project `ref` are features of dbt Cloud Enterprise, currently in Closed Beta. To access these features while they are in beta, please contact your account team at dbt Labs.
+
+**Prerequisites:** In order to add project dependencies and resolve cross-project `ref`, you must:
+- Have the feature enabled (speak to your account team)
+- Be using dbt v1.6 in **both** the upstream ("producer") project and the downstream ("consumer") project
+- Have a deployment env in the upstream ("producer") project [that is set to be your production environment](docs/deploy/deploy-environments#set-as-production-environment-beta)
```

Comment:
add / to prevent broken link

```suggestion
- Have a deployment environment in the upstream ("producer") project [that is set to be your production environment](/docs/deploy/deploy-environments#set-as-production-environment-beta)
```